dmutils/api_stubs.py

```diff
@@ -28,6 +29,41 @@ def lot(lot_id=1, slug="some-lot", name=None, allows_brief=False, one_service_li
     }
 
 
+def __as_a_service_lots():
+    return [
+        lot(lot_id=1, slug='saas', name='Software as a Service'),
+        lot(lot_id=2, slug='paas', name='Platform as a Service'),
+        lot(lot_id=3, slug='iaas', name='Infrastructure as a Service'),
+        lot(lot_id=4, slug='scs', name='Specialist Cloud Services')
+    ]
+
+
+def __cloud_lots():
+    return [
+        lot(lot_id=9, slug='cloud-hosting', name='Cloud hosting'),
+        lot(lot_id=10, slug='cloud-software', name='Cloud software'),
+        lot(lot_id=11, slug='cloud-support', name='Cloud support')
+    ]
+
+
+g_cloud_4_lots = __as_a_service_lots
+g_cloud_5_lots = __as_a_service_lots
+g_cloud_6_lots = __as_a_service_lots
+g_cloud_7_lots = __as_a_service_lots
+g_cloud_8_lots = __as_a_service_lots
+g_cloud_9_lots = __cloud_lots
+g_cloud_10_lots = __cloud_lots
+
+
+def dos_lots():
+    return [
+        lot(lot_id=5, slug='digital-outcomes', name='Digital outcomes', allows_brief=True),
```

Comment:
Could add `one_service_limit=True` for outcomes, specialists and user-research-participants?

Reply:
👍